domain-ee/ee-ep-merge-app/integration/test_merge_request.py

```diff
@@ -197,11 +198,11 @@ async def test_completed_error_at_update_claim_contentions(self,
                                                                get_claim_endpoint: MqEndpoint,
```

Comment:
Minor nit: consider renaming test to `test_completed_error_at_create_claim_contentions` in line with endpoint change.